app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+# app/models/post.rb
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+
+  def update_user_posts_counter
```

Comment:
- [ ] Hey, great job on the code so far! 🚀 However, there's a minor hiccup that doesn't align with the project requirements. According to the specs, after adding a post, the post_counter in the users table should autoincrement. 📈 Could you please address this by implementing the necessary changes? Thanks a bunch! 🙌

![Screenshot from 2024-01-08 13-35-03](https://github.com/DevKaranJ/Blog/assets/112261237/0ece69e3-4ba8-4f41-8a43-88c1288f5ba9)


```suggestion
after_save :update_post_counter

 private
  def update_post_counter
    author.increment!(:posts_counter)
  end  
```